src/deluge/gui/menu_item/value_scaling.h

```diff
@@ -72,6 +74,31 @@ int32_t computeCurrentValueForStandardMenuItem(int32_t value);
  */
 int32_t computeFinalValueForStandardMenuItem(int32_t value);
 
+/** Scales INT32_MIN-INT32_MAX range to 0-50 for display.
+ *
+ * This roundtrips with the final value math despite being not being it's
+ * proper inverse.
+ *
+ * Thin wrapper for clarity.
+ */
+inline int32_t computeCurrentValueForSemiStandardMenuItem(int32_t value) {
+	return computeCurrentValueForStandardMenuItem(value);
+}
+
+/** Scales 0-50 range to INT32_MIN-(INT32_MAX-45) for storage and use.
+ *
+ * It is unclear when this is used intentionally, and when by accident.
+ *
+ * - arpeggiator::midi_cv::Gate: possibly so that gate
+ *   will momentarily go down even at 50: the values produced create a 2.5ms
```

Comment:
this is enforced by the CV engine so it shouldn't matter. I'm inclined to think this is always an accident

Reply:
I'll update the comment, and leave the "how can we make things more consistent" for later.

Reply:
Worth testing but I'd be surprised if it even changes sounds. It's a 2e-8% difference

Reply:
Yeah, no measurable difference for gate.